### PR TITLE
Update release notes for v1.0

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+- Fixed a bug where `podman inspect -f '{{ json .Config }}'` on images would not output anything (it now prints the image's config)
+- Fixed a bug where `podman rmi -fa` displayed the wrong error message when trying to remove images used by pod infra containers
+
+### Misc
+- Rootless containers now unconditionally use postrun cleanup processes, ensuring resources are freed when the container stops
+- A new version of Buildah is included for `podman build`, featuring improved build speed and numerous bugfixes
+- Pulling images has been parallelized, allowing individual layers to be pulled in parallel
+- The `podman start --attach` command now defaults the `sig-proxy` option to `true`, matching `podman create` and `podman run`
+- The `podman info` command now prints the path of the configuration file controlling container storage
+- Added `podman list` as an alias for `podman ps`, and `podman container ps` as an alias for `podman container list`
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: 'podman container ls' no?  I don't think it's list, rather ls.

Comment (TomSweeneyRedHat) [reply RELEASE_NOTES.md]: ditto 'podman ls' me thinks.

Comment (TomSweeneyRedHat) [reply RELEASE_NOTES.md]: And I don 't see a 'ls' or 'list' command on the https://github.com/containers/libpod/blob/master/docs/podman.1.md page.  Should one be added?

Comment (mheon) [reply RELEASE_NOTES.md]: Actually we support both... `podman ps` == `podman ls` == `podman list`

**Comment by TomSweeneyRedHat** [comment]
LGTM once the alias note is cleaned up.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply RELEASE_NOTES.md]: OK, that's fine then, but there's no 'podman ls' or 'podman list' on the podman man page, just 'podman ps'.  Might want to circle back to add them.

**Pull request merged**